src/containers/TemplateBuilder/shared/OperationContextHelpers.tsx

```diff
@@ -269,7 +269,7 @@ export const importTemplate: ImportTemplateHelper =
       data.append('file', file)
 
       const result = await safeFetch(
-        getServerUrl('snapshot', { action: 'upload', name: snapshotName }),
```

Comment:
No need to include name as a parameter -- we'll use the filename of the upload as the name (since the file was named as such when downloading)